test/sqlancer/mzcompose.py

```diff
@@ -81,18 +81,18 @@ def kill_sqlancer_with_delay() -> None:
         "materialize",
         "--oracle",
         args.oracle,
-        capture=True,
-        capture_stderr=True,
         check=False,
+        detach=True,
+        capture=True,
     )
+    container_id = result.stdout.strip()
 
-    if result.returncode not in (0, 130):
```

Comment:
So we are no longer converting certain error codes to test failures?

Reply:
Correct. This hasn't found anything and I'm not sure if there is a way to do so after using `docker compose run --detach`.

Reply:
ok fair enough.